src/app/frontend/ingress/list/card.html

```diff
@@ -47,5 +47,13 @@
       </div>
       <div ng-if="::!$ctrl.ingress.objectMeta.creationTimestamp">-</div>
     </kd-resource-card-column>
+    <kd-resource-card-column class="kd-row-layout-column kd-icon-column">
+      <kd-resource-card-menu>
+        <kd-resource-card-delete-menu-item resource-kind-name="Ingress">
```

Comment:
This is a display name. Should be localized.

Reply:
@floreks Okay, I will fix this issue in next commit.